sympy/integrals/intpoly.py

```diff
@@ -100,6 +100,16 @@ def polytope_integrate(poly, expr=None, *, clockwise=False, max_degree=None):
 
     if max_degree is not None:
         result = {}
+        if expr is not None:
+            f_expr = []
+            for exp in expr:
```

Comment:
how do you know you have a list for `expr` at this point?

Also, please use `e` instead of `exp` (which looks like `exp`-function, e.g. `exp(3)`)

Reply:
What I understood was that either there can be a single expression as input which will be handled by` max_degree=None ` condition, or there could be a list .On master the first condition check for `max_degree is not None ` case is ` if not isinstance(expr, list) and expr is not None:` .This , along with the docstring/ online documentation made me assume that list input is expected whenever `max_degree` is not None

Reply:
I will make the variable name changes you requested .Thanks @smichr !

Reply:
I think once this is resolved ,this pr should be ready to go in .Would appreciate any relevant comments .Thanks!